spec/src/main/asciidoc/configuration.asciidoc

```diff
@@ -257,6 +257,22 @@ Reloading the Public Key from the location at runtime as well as the frequency o
 reloading is beyond the scope of this specification and any such feature should be
 considered vendor-specific.
 
+#### `mp.jwt.token.header`
+
+The `mp.jwt.token.header` configuration property allows to set up the header which is expected to contain a JWT token.
+
+This configuration can either be `Authorization` (default), or `Cookie`. If a different configuration value is supplied,
+the configuration will be ignored and fallback to the default `Authorization`.
+
+Additional configuration options might be available in future versions of MP-JWT.
```

Comment:
+1 to this line, though I'd consider at the moment only refer to `mp.jwt.token.header`: `Other headers may be supported in future versions of MP-JWT` since it is not clear what other configuration options would be about.

Reply:
@radcortez Oh may be `Other headers and authentication schemes may be supported in future versions of MP-JWT` if you wanted to imply the schemes ?

Reply:
Sure.